src/commands/local.ts

```diff
@@ -25,9 +25,11 @@ export const local = cli()
       verbose.enabled = true;
     }
 
+    verbose.log("Starting local inference with options: ", opts);
+
     const prompt = await llama({
       model: opts.model,
-      disableGPU: opts.disableGPU,
+      disableGpu: opts.disableGpu,
```

Comment:
Should the TypeScript typing have pointed out this issue earlier?

Reply:
Yes, but unfortunately the `commander` package just passes `any` as the options. `yargs` was better in making the options strongly typed, although I found the `commander` package a bit more ergonomic to use. I'm thinking about changing away from `commander` though, exactly because this should have been a compiler-caught bug.

Reply:
Ah true; yeah, that makes sense.